controllers/review.controller.js

```diff
@@ -26,8 +26,14 @@ exports.updateReview = (req, res, next) => {
 }
 
 exports.getAllReviews = (req,res,next) => {
-    fetchAllReviews().then(reviews => {
-        res.status(200).send({reviews})
+    fetchAllReviews(req.query).then(reviews => {
+        if(!reviews.length){
```

Comment:
What about if the category exists - but there are no associated reviews... yet. 
You could easily search for a something like 'family games' and not get any results - but this might not be an error. 
Think of a way of checking the category exists...